src/citrine/resources/design_execution.py

```diff
@@ -0,0 +1,191 @@
+"""Resources that represent both individual and collections of workflow executions."""
+from functools import lru_cache, partial
+from typing import Optional, Iterable, Union
+from uuid import UUID
+
+from citrine._rest.collection import Collection
+from citrine._rest.resource import Resource
+from citrine._serialization import properties
+from citrine._session import Session
+from citrine.informatics.design_candidate import DesignCandidate
+from citrine.resources.response import Response
+from citrine.informatics.scores import Score
+
+
+class DesignExecution(Resource['DesignExecution']):
+    """The execution of a DesignWorkflow.
+
+    Parameters
+    ----------
+    uid: str
+        Unique identifier of the workflow execution
+    project_id: str
+        Unique identifier of the project that contains the workflow execution
+    workflow_id: str
+        Unique identifier of the workflow that was executed
+
+    """
+
+    _response_key = None
+
+    uid: UUID = properties.UUID('id', serializable=False)
+    workflow_id = properties.UUID('workflow_id', serializable=False)
+    status = properties.Optional(properties.String(), 'status', serializable=False)
```

Comment:
I think we're missing `status_description` (the minor status).

Reply:
fixing